app/javascript/actionsPanel/initializeActionsPanelToggle.js

```diff
@@ -17,7 +17,9 @@ export function initializeActionsPanel(user, path) {
   `;
 
   function toggleModActionsMenu() {
-    document.getElementById('mod-actions-menu-btn-area').classList.remove('hidden');
```

Comment:
I don't know how this has sneaked in here, but it's just formatting changes so 🤷‍♀️

Reply:
UGH...